### PR TITLE
X-API-KEY guide

### PR DESCRIPTION
**Pull request by sdelamo**
[Micronaut Security API Key.pdf](https://github.com/micronaut-projects/micronaut-guides/files/10686259/Micronaut.Security.API.Key.pdf)

### CODE REVIEW

File: src/docs/common/callouts/callout-each-property.adoc [hunk 1]
```diff
@@ -0,0 +1 @@
+The `@EachProperty` annotation creates a `ConfigurationProperties` bean for each sub-property within the given property.
```
Comment (alvarosanchez) [inline src/docs/common/callouts/callout-each-property.adoc]:
```suggestion
The `@EachProperty` annotation creates a `ConfigurationProperties` bean for each sub-property within the given name.
```

File: src/docs/common/callouts/callout-principal.adoc [hunk 2]
```diff
@@ -0,0 +1 @@
+You can bind java.security.Principal as a method’s parameter in a controller.
```
Comment (alvarosanchez) [inline src/docs/common/callouts/callout-principal.adoc]:
```suggestion
You can bind `java.security.Principal` as a method’s parameter in a controller.
```

**Review by alvarosanchez — APPROVED** [review]

File: src/docs/common/callouts/callout-each-property.adoc (hunk 1 above)
Comment (sdelamo) [reply src/docs/common/callouts/callout-each-property.adoc]: @alvarosanchez I think we should change it in core as well https://docs.micronaut.io/latest/guide/#eachProperty. Thoughts?

Comment (alvarosanchez) [reply src/docs/common/callouts/callout-each-property.adoc]: Fine to me. It's just to not have the word "property" 3 times.

**Pull request merged**